### PR TITLE
Allow to deploy master branches

### PR DESCRIPTION
**Pull request by sileht**
This change allow to run a custom version of Gnocchi, collectd-gnocchi
and the grafana plugin by setting environment variables.

It also put as example for master branches of each projects.

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now f85919ea39e432f96c39c45449a36d0b47ef0ad1

**Comment by sileht** [comment]
Pull-request updated, HEAD is now bb8b90dac30a0edf11101571428657e41873f47c

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 47011c7e008ad9c58c7ff0eb7c00db0bb8a0aa99

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 979212ff345d6d3e15a6207dd760a51ee0300ff1

**Comment by sileht** [comment]
Pull-request updated, HEAD is now c1e13d660992f806381fc5d9524fbe976ec6ed1f

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 4a21a1b6529e6268dbf5a6940a10d90345b77685

**Comment by sileht** [comment]
Pull-request updated, HEAD is now a0ccfe2bb7a6b78bd1bd8667fde715617bc8069d

File: devel.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (jd) [inline devel.env]: empty line?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now fb44238c6c17fc569a313ad2ce7b8d91139dea90

**Pull request merged**